pkg/pillar/devicenetwork/devicenetwork.go

```diff
@@ -255,6 +250,7 @@ func MakeDeviceNetworkStatus(globalConfig types.DevicePortConfig, oldStatus type
 			&globalStatus.Ports[ix])
 		if err != nil {
 			errStr := fmt.Sprintf("GetNetworkProxy failed %s", err)
+			// Clobbers ErrorAndTime from above
 			globalStatus.Ports[ix].Error = errStr
 			globalStatus.Ports[ix].ErrorTime = time.Now()
```

Comment:
Not directly related.. but should this be - 
       globalStatus.Ports[ix].ErrorAndTime.SetOrAppendError(types.ErroAndTimeNow(errStr)) ?

This would append the errors ?

I guess the case here is:

1) DPC has an error detected during parse config.
  -- Do we skip any further tests on the port till that error is Cleared by next config change?
  -- Or - if we deem the error to be minor - and we can still proceed - append to that error as above.

Reply:
@kalyan-nidumolu I flagged that on your PR a few days back (post pull); the PR which replaced/removed ParseError.
Note that appending errors is in general the wrong answer if you need to be able to remove one of the errors (which is normal as part of implementing eventual consistency).
So can you please file a tracker story on the ParseError vs. error from testing and work on it?